index.html

```diff
@@ -20,12 +20,14 @@
       </li>
       <li class="flex-item"></li>
       <li class="flex-item">
-        <a href="#"><img src="./images/ic_menu.png" alt="ic_menu" class="ic-menu" srcset=""></a>
+        <nav>
+          <a href="#"><img src="./images/ic_menu.png" alt="ic_menu" class="ic-menu" srcset=""></a>
+        </nav>
       </li>
     </ul>
   </header>
   <main class="flex-main">
-    <p class="name">Fickry <br>Bil Iman</p>
+    <h1 class="name">Fickry <br>Bil Iman</h1>
     <div class="greetings">
       <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.</p>
       <p>If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
```

Comment:
- [ ] [OPTIONAL] Please,  I suggest you increase the font weight and of this text, to make it more pixel perfect 🙂.

- Your design
<img width="157" alt="wrong" src="https://user-images.githubusercontent.com/41163571/219457357-4efc4b65-e204-4d5b-8956-87bd8a1e516b.png">

- Figma design
<img width="281" alt="right" src="https://user-images.githubusercontent.com/41163571/219457398-3994a83d-eda6-4471-881d-6085194feec4.png">

Reply:
Thank you, i will increase it.